pkg/satellite/stream/stream.go

```diff
@@ -88,7 +92,7 @@ func (ss *satelliteStream) Send(payload []byte) error {
 	return ss.stream.Send(&req)
 }
 
-// Close closes the stream.
```

Comment:
I think the former one is the normal go convention? You always start with the method name